src/providers/AccessControlProviderCustom.ts

```
@@ -27,69 +22,115 @@ export const AccessControlProviderCustom: AccessControlProvider = {
             return { can: true };
 
           case 'create':
+            const canCreate = [UserRole.SUPERADMIN, UserRole.TM, UserRole.CNBM].includes(userRole);
             return {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)